QA PRs checking process for PL 0.40 during feature freeze (#6775)
PennyLaneAI/pennylane: doc/releases/changelog-0.40.0.md

  ```

* Default implementations of `Device.setup_execution_config` and `Device.preprocess_transforms`

>   are added to the device API for devices that provide a TOML configuration file and thus have

Reviewer: Something has always bothering me about "thus" - totally optional!
```suggestion
  are added to the device API for devices that provide a TOML configuration file, thereby having
```

  [(#6567)](https://github.com/PennyLaneAI/pennylane/pull/6567)

* The `diagonalize_measurements` transform no longer raises an error for unknown observables. Instead,

>   they are left un-diagonalized, with the expectation that observable validation will catch any un-diagonalized

Reviewer 2: Why hyphen? I thought no hyphen version is more conventional?

Author: It was a Grammarly suggestion (I usually trust it for this since I am not an English-native speaker)
